src/bindings/python/fluxacct/accounting/formatter.py

```diff
@@ -246,21 +246,26 @@ def construct_parsable_hierarchy(cur, bank, hierarchy, indent=""):
                 if users:
                     for user in users:
                         hierarchy += (
-                            f"{indent} {bank}|{user[0]}|{user[1]}|{user[2]}|{user[3]}\n"
+                            f"{indent} {bank}|{user['username']}|{user['shares']}|"
+                            f"{user['job_usage']}|{user['fairshare']}\n"
                         )
+
             else:
                 # continue traversing the hierarchy
                 for sub_bank in sub_banks:
-                    hierarchy += f"{indent} {str(sub_bank[0])}||{str(sub_bank[1])}|{str(sub_bank[2])}\n"
+                    hierarchy += (
+                        f"{indent} {str(sub_bank['bank'])}||{str(sub_bank['shares'])}|"
+                        f"{str(sub_bank['job_usage'])}\n"
+                    )
                     hierarchy = construct_parsable_hierarchy(
-                        cur, sub_bank[0], hierarchy, indent + " "
+                        cur, sub_bank["bank"], hierarchy, indent + " "
                     )
 
             return hierarchy
 
         # construct a hierarchy string starting with the bank passed in
         hierarchy = "Bank|Username|RawShares|RawUsage|Fairshare\n"
-        hierarchy += f"{self.rows[0][1]}||{str(self.rows[0][4])}|{str(round(self.rows[0][5], 2))}\n"
+        hierarchy += f"{self.rows[0]['bank']}||{str(self.rows[0]['shares'])}|{str(round(self.rows[0]['job_usage'], 2))}\n"
```

Comment:
Use double quotes for consistency?

Reply:
I don't think I can use double quotes here, right? Because I already wrapped the f-string in double quotes. But I could switch those (i.e use single quotes for the f-string and double quotes for the keys in the curly braces) if you'd like? I'll defer to your preference.

Reply:
Oh yeah you're right, I was totally missing that it was inside double quotes already.